nature/models.py

```diff
@@ -83,13 +85,23 @@ class FeatureQuerySet(ProtectionLevelQuerySet):
     """
     QuerySet class For Feature model
     """
+    def for_office_hki(self):
+        links = FeatureLink.objects.filter(protection_level__gte=PROTECTION_LEVELS['OFFICE'])
+        prefetch = Prefetch('links', queryset=links)
+        return super().for_office_hki().prefetch_related(prefetch)
+
     def for_office(self):
```

Comment:
According to how class inheritance works, this method actually needs prefetching too.

Reply:
Thanks for feedback. Edited as requested.